.github/actions/vital-throw-message/entrypoint.sh

```diff
@@ -4,9 +4,7 @@ cd "$GITHUB_WORKSPACE"
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-go get -d -v ./scripts
-
-go run ./scripts/lint-throw.go $(find autoload/vital/__vital__/ | grep -e '\.vim$') \
+go run -mod=mod ./scripts/lint-throw.go $(find autoload/vital/__vital__/ | grep -e '\.vim$') \
```

Comment:
```suggestion
go run -mod=mod /scripts/lint-throw.go $(find autoload/vital/__vital__/ | grep -e '\.vim$') \
```

すみません、ここ自分の初期作成ミスがあり、上のようにしないと動いていませんでした。

see https://github.com/tsuyoshicho/vital-codec/pull/241
(より整理したもの+テストNGのPR)